bundle/tests/scorecard/config.yaml

```diff
@@ -12,59 +12,38 @@ stages:
     labels:
       suite: basic
       test: basic-check-spec-test
-    storage:
```

Comment:
@saumeya I think this is not intentional ? What is the version of Operator-SDK that you are using ?

Reply:
thanks, yes I am using an old one `operator-sdk version: "v1.10.1"`